tests/cli/conftest.py

```diff
@@ -2,12 +2,13 @@
 
 from subprocess import check_call
 
-from _pytest.tmpdir import TempdirFactory
 from typing import Callable, Text
 import pytest
 import shutil
 import os
-from _pytest.pytester import Testdir, RunResult
+
+from _pytest.legacypath import TempdirFactory, Testdir
+from _pytest.pytester import RunResult
```

Comment:
The leading underscore is fishy. Docs look like we should be able to import directly from pytest, i.e. `from pytest import TempdirFactory`